apps/public-docsite-v9/src/Concepts/Motion/CustomMotion.stories.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from 'react';
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32437&commitId=dc7e87e80fbdd1d9f95e6faf3a3cf84c1d8d9a70&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|3| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|19| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|975| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|2244| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>